src/bootman/sysconfig.c

```diff
@@ -44,6 +46,14 @@ int cbm_get_fstype(const char *boot_device)
         const char *data;
         int ret = 0;
 
+        /* When running tests, assume FATFS */
```

Comment:
This comment seems misleading, probably just should be when running tests try to use the CBM_TEST_FSTYPE env as the type.

Reply:
changed to: "Test suite will set the CBM_TEST_FSTYPE env var and inform the wanted fstype" - thanks.